datamegh/__init__.py

```diff
@@ -13,7 +13,7 @@
     "warning": WARNING,
     "critical": CRITICAL,
 }
-config = {
+config: object = {
```

Comment:
```suggestion
config: Dict = {
```

Reply:
There was 2 errors. 1 got fixed by adding object to it.

Reply:
OK then.